src/lib/y2storage/guided_proposal.rb

```diff
@@ -271,7 +271,7 @@ def remove_empty_partition_tables(devicegraph)
     def candidate_devices_with_empty_partition_table(devicegraph)
       device_names = settings.candidate_devices
       devices = device_names.map { |n| devicegraph.find_by_name(n) }
```

Comment:
NP: maybe using `device_names.map { |n| devicegraph.find_by_name(n) }.compact` would be easier to read.